docs/src/inline-asm.md

```diff
@@ -0,0 +1,15 @@
+# Inline Assembly
+Rust-GPU has support for inline SPIR-V assembly. In addition the backend
+provides several conveniences for writing inline assembly that are documented
+below. For more information on specific instruction behaviour and syntax, please
+refer to the [SPIR-V specification][spec].
+
+[spec]: https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html
+
+### Additional syntax
+
+| Syntax | Description |
+| ------ | ----------- |
+| `%name` | Used to refer to an abstract ID of a type. |
```

Comment:
`%name` refers to any abstract ID, not just types. Might be good to point out that we do the same thing as `spirv-as` in that every unique `%name` generates a new, unrelated ID, even if an integer is written.